status/status.go

```diff
@@ -92,10 +99,20 @@ func FromError(err error) (s *Status, ok bool) {
 	}
 	type grpcstatus interface{ GRPCStatus() *Status }
 	if gs, ok := err.(grpcstatus); ok {
+		if gs.GRPCStatus() == nil {
+			// Error has a status of OK. There is no sensible behavior for this, so map it to Unknown and discard the
```

Comment:
Why does the comment say that the error has a status of `OK` when the status is `nil`? Do we need to say that in the comment?

Also, nit: could you please wrap the comment lines at 80-cols.

Reply:
Yes, I clarified the comment -- it's not the error that is nil, it is its status, and nil status has codes.OK. I also wrapped to 80 cols (assuming 8 space for a tab).